PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ServiceBusReceiverAsyncClientCheck.java

```diff
@@ -0,0 +1,78 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.JavaElementVisitor;
+import com.intellij.psi.PsiTypeElement;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+
+/**
+ * This class extends the LocalInspectionTool to check for the use of ServiceBusReceiverAsyncClient
+ * in the code and suggests using ServiceBusProcessorClient instead.
+ */
+public class ServiceBusReceiverAsyncClientCheck extends LocalInspectionTool {
+
+    public static final String configFileName = "META-INF/ruleConfigs.json";
```

Comment:
```suggestion
    public static final String RULE_CONFIGURATION= "META-INF/ruleConfigs.json";
```
https://www.baeldung.com/java-constants-good-practices